en/docs/chapter_sorting/quick_sort.md

```diff
@@ -1,12 +1,12 @@
 # Quick sort
 
-<u>Quick sort</u> is a sorting algorithm based on the divide and conquer strategy, known for its efficiency and wide application.
+<u>Quick sort</u> is a sorting algorithm based on the divide-and-conquer strategy, known for its efficiency and wide application.
 
-The core operation of quick sort is "pivot partitioning," aiming to: select an element from the array as the "pivot," move all elements smaller than the pivot to its left, and move elements greater than the pivot to its right. Specifically, the pivot partitioning process is illustrated in the figure below.
+The core operation of quick sort is "pivot partitioning," which aims to select an element from the array as the "pivot" and move all elements less than the pivot to its left side, while moving all elements great than the pivot to its right side. Specifically, the process of pivot partitioning is illustrated in the figure below.
 
-1. Select the leftmost element of the array as the pivot, and initialize two pointers `i` and `j` at both ends of the array.
-2. Set up a loop where each round uses `i` (`j`) to find the first element larger (smaller) than the pivot, then swap these two elements.
-3. Repeat step `2.` until `i` and `j` meet, finally swap the pivot to the boundary between the two sub-arrays.
+1. Select the leftmost element of the array as the pivot, and initialize two pointers `i` and `j`, to point to the two ends of the array respectively.
+2. Set up a loop where each round uses `i` (`j`) to search for the first element larger (smaller) than the pivot, then swap these two elements.
+3. Repeat step `2.` until `i` and `j` meet, finally swap the pivot to the boundary between the two subarrays.
```

Comment:
`sub-arrays` -> `subarrays` breaks the consistency.

Reply:
Ok, I have changed them back